src/modules/displaymanager/main.py

```diff
@@ -403,6 +401,11 @@ def run():
     # setup lightdm
     if "lightdm" in displaymanagers:
         if have_dm("lightdm", root_mount_point):
+            global lightdm_conf_path
```

Comment:
Why is this a global? The rest of the diff suggests this variable is local to run()